thoth/package_extract/image.py

```diff
@@ -546,6 +546,47 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            output = f.read()
+            lines = output.split()
+            if len(lines) != 3:
+                _LOGGER.info(
+                    "Unable to detect CUDA version from usr/local/cuda/version.txt output: %r",
+                    output,
+                )
+            else:
+                res["/usr/local/cuda/version.txt"] = lines[-1]
+
+    # gathering version from nvcc command
+    nvcc_path = glob.glob(path + '/usr/local/cuda-*/bin/nvcc')
+    if len(nvcc_path) > 0:
+        st = os.stat(nvcc_path[0])
+        os.chmod(nvcc_path[0], st.st_mode | stat.S_IEXEC)
+        result = run_command("{} --version".format(nvcc_path[0]), raise_on_error=False)
+        if result.return_code != 0:
+            _LOGGER.info(
+                "Unable to detect CUDA version - nvcc returned non-zero version: %s",
+                result.to_dict(),
+            )
+        else:
+            lines = result.stdout.splitlines()
+            version_info = lines[-1].split(",")
+            if len(version_info) != 3:
+                _LOGGER.debug(
+                    "Unable to detect CUDA version from nvcc output: %r", result.stdout
+                )
+            else:
+                res["nvcc_version"] = version_info[-1].strip()[1:]
```

Comment:
So, we are stripping and to get `10.1.243` from `V10.1.243`, so we start from `[1:]`.

Reply:
What about checking if the version really starts with `V`?